### PR TITLE
docs: add uups upgrade example

### PR DESCRIPTION
**Pull request by MCarlomagno**
# Summary
Added example to showcase a simple UUPS contract upgrade using Defender

### CODE REVIEW

File: examples/upgrade-contract-uups/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+const uupsOwnableAbi = require('./abis/UUPSOwnable.json');
+
+async function main() {
+  const client = new Defender({
+    apiKey: process.env.API_KEY,
+    apiSecret: process.env.API_SECRET,
+  });
+
+  // Get approval process for deployment on Sepolia
+  const config = await client.deploy.getUpgradeApprovalProcess('sepolia');
```
Comment (CoveMB) [inline examples/upgrade-contract-uups/index.js:14]: nit: Do we need to log the approval processes if it is not used?

**Review by CoveMB — APPROVED** [review]

File: examples/upgrade-contract-uups/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+## Defender SDK upgrade
+
+This example showcases a simple contract upgrade using UUPS + ownable.
+
+### Steps to run this example
+
+1. Create a deployment environment in sepolia network, (https://docs.openzeppelin.com/defender/v2/tutorial/deploy#environment_setup)[see how]
```
Comment (shahnami) [inline examples/upgrade-contract-uups/README.md:7]:
```suggestion
1. Create a deployment environment for the sepolia network, (https://docs.openzeppelin.com/defender/v2/tutorial/deploy#environment_setup)[see how]
```

File: examples/upgrade-contract-uups/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+```js
+const upgrade = await client.deploy.upgradeContract({
+  proxyAddress: '0x3a...ad7', // erc1967 proxy address
+  newImplementationAddress: '0x484...99', // address of the new implementation contract
+  newImplementationABI: JSON.stringify(uupsOwnableAbi), // The ABI of the new implementation (must implement UUPS standard)
+  network: 'sepolia',
+});
+```
+
+6. Go to your defender dashboard to see the status of your upgrade.
```
Comment (shahnami) [inline examples/upgrade-contract-uups/README.md:22]:
```suggestion
6. Go to your Defender dashboard to see the status of your upgrade.
```

**Review by shahnami — APPROVED** [review]
Lgtm!

**Pull request merged**